src/navigators/ConnectSignUpStackNavigator.js

```diff
@@ -3,27 +3,29 @@
  * to ensure that the user doesn't see the Connect screen until the
  * Connect sign up process has concluded.
  */
- 
-import React from 'react';
-import { StackNavigator } from 'react-navigation';
-import ConnectScreen from '../screens/ConnectScreen';
-import ConnectSignUpScreen from '../screens/ConnectSignUpScreen';
-import ProfileScreen from '../screens/ProfileScreen';
 
+import React from "react";
+import { createStackNavigator } from "react-navigation";
+import ConnectScreen from "../screens/ConnectScreen";
+import ConnectSignUpScreen from "../screens/ConnectSignUpScreen";
+import ProfileScreen from "../screens/ProfileScreen";
 
-const ConnectSignUpStackNavigator = StackNavigator({
-  Connect: {
-    screen: ConnectScreen,
+const ConnectSignUpStackNavigator = createStackNavigator(
+  {
```

Comment:
This is one of the main changes though cause newer react navigation usage is pretty different from how we did things before